megalinter/descriptors/kubernetes.megalinter-descriptor.yml

```
@@ -113,6 +113,9 @@ linters:
         - libc6-compat
         - libstdc++
       dockerfile:
-        - |-
+        - |
+          # renovate: datasource=github-tags depName=kubescape/kubescape
+          ARG KUBESCAPE_VERSION=3.0.14
```

```suggestion
        - |-
          # renovate: datasource=github-tags depName=kubescape/kubescape
          ARG KUBESCAPE_VERSION=3.0.14
```
The tack removes the empty trailing line and the pipe keeps new lines (yaml definition)